src/SDKs/Batch/Management/Management.Batch.Tests/Management.Batch.Tests.csproj

```diff
@@ -13,6 +13,7 @@
   <ItemGroup>
     <PackageReference Include="Microsoft.Azure.Management.ResourceManager" Version="[1.1.3-preview,2.0.0)" />
     <ProjectReference Include="..\Management.Batch\Microsoft.Azure.Management.Batch.csproj" />
+    <PackageReference Update="Microsoft.NET.Test.Sdk" Version="15.3.0" />
```

Comment:
Why are you changing batch.Tests projects?

Reply:
it is accidently changed. updated. thanks!